integration_tests/models/plugins/snowflake/snowflake_external.yml

```diff
@@ -20,6 +20,11 @@ sources:
             data_type: varchar(64)
           - name: email
             data_type: varchar(64)
+          - name: email domain
+            data_type: varchar(64)
+            quote: true
+            identifier: email
+            expression: split_part(value:email::VARCHAR, '@', 2)
```

Comment:
When you add the column here, it becomes a part of a YAML anchor, `&cols-of-the-people` which is used in almost all the other tests which caused them to fail.

check out what i did in #290 to actually test the change in #257. I think you could follow a similar pattern to get it passing